edx_proctoring/tests/test_views.py

```diff
@@ -4961,6 +4967,7 @@ def test_add_bulk_allowance_non_staff_user(self):  # pylint: disable=invalid-nam
         exam_list = [exam1.id, exam2.id]
 
         allowance_data = {
+            'course_id': self.course_id,
```

Comment:
I think we should also remove course_id from the kwargs since that's not how it's expected to come in